### PR TITLE
[ENG-622] Add contact source to the global search results modal

### PR DESCRIPTION
**Pull request by scottshipman**

### CODE REVIEW

File: Views/SubscribedEvents/Search/global.html.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+       data-toggle="ajax">
+        <span><?php echo $view['translator']->trans('mautic.core.search.more', ['%count%' => $remaining]); ?></span>
+    </a>
+<?php else: ?>
+    <a href="<?php echo $view['router']->generate(
+        'mautic_contactsource_action',
+        ['objectAction' => 'view', 'objectId' => $source->getId()]
+    ); ?>" data-toggle="ajax">
+        <span><?php echo $source->getName(); ?></span>
+        <?php
```
Comment (dean-martin) [inline Views/SubscribedEvents/Search/global.html.php]: Extra `<?php ?>` ?

Comment (westonwatson) [reply Views/SubscribedEvents/Search/global.html.php]: this has markup (html) interlaced with PHP, so there will be multiple <?php> tags, you can see that the previous php tag was closed (?>) right before the "</span>" on line 22.

Comment (dean-martin) [reply Views/SubscribedEvents/Search/global.html.php]: I get that, but that's an empty <?php ?> with nothing inside it though.... No <?php else: ?> or <?php echo "" ?>. Just a PHP tag with nothing inside

Comment (heathdutton) [reply Views/SubscribedEvents/Search/global.html.php]: An empty php tag just to make sure that PHP feels included.

**Pull request merged**